src/CommonDBTM.php

```diff
@@ -4720,10 +4720,9 @@ public static function getSpecificValueToDisplay($field, $values, array $options
 
         switch ($field) {
             case '_virtual_datacenter_position':
-                /** @var CommonDBTM|\Glpi\Features\DCBreadcrumb $static */
                 $static = new static();
                 if (method_exists($static, 'getDcBreadcrumbSpecificValueToDisplay')) {
-                    //@phpstan-ignore-next-line
+                    //FIXME phpstan-ignore-next-line
```

Comment:
Is probably useless since addition of  `/** @var CommonDBTM|\Glpi\Features\DCBreadcrumb $static */`.

Reply:
Maybe not, not sure we can force casting to a trait.